### ISSUE TITLE
Colour contrast issue with code examples

### ISSUE THREAD

**Issue by ahosgood** [nationalarchives/design-system#2]
The colour contrast ratio of code examples using the light theme don't meet the minimum contrast ratio of 4.5:1 for small text.

This is a failure of [WCAG success criterion 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum).

**Comment by ahosgood** [comment]
Consider using one of these: https://github.com/PrismJS/prism-themes
